src/BarChart/RenderStackBars.tsx

```diff
@@ -293,17 +326,15 @@ const RenderStackBars = (props: Props) => {
                 borderRadius ??
                 stackBorderBottomRightRadius ??
                 stackBorderRadius,
-              overflow: 'hidden',
```

Comment:
@christophby Was this change intentional? `stackBorderRadius` is not working after removing this.

Reply:
So this change is needed. I have merged it. To let `stackBorderRadius` work, I have added this property conditionally.

Reply:
Yes, it is. Thanks a lot